ibis/backends/flink/registry.py

```diff
@@ -219,6 +268,52 @@ def _floor_divide(translator: ExprTranslator, op: ops.Node) -> str:
     return f"FLOOR(({left}) / ({right}))"
 
 
+def _array_index(translator: ExprTranslator, op: ops.arrays.ArrayIndex):
+    table_column = op.arg
+    index = op.index
+
+    table_column_translated = translator.translate(table_column)
+    index_translated = translator.translate(index)
+
+    return f"{table_column_translated} '[' {index_translated} ']'"
+
+
+def _day_of_week_index(translator: ExprTranslator, op: ops.Node) -> str:
+    arg = translator.translate(op.args[0])
```

Comment:
Prefer `op.arg` (and in general accessing the argument by name) over accessing `op.args[index]`

Reply:
Thanks, changed to `op.arg`.